addon/components/o-s-s/input-container.ts

```diff
@@ -1,9 +1,13 @@
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 
+type MessageType = 'error' | 'alert' | 'success';
```

Comment:
nit: whatabout `FeedbackMessageType` ? 🤔

Reply:
I tried many configurations to make this works and stopped to the first one working, but it seems 50% works too indeed